alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/accessor/DefaultDescriptorGlobalConfigUtility.java

```diff
@@ -108,7 +108,7 @@ public FieldModel update(Long id, FieldModel fieldModel) throws AlertException {
             configurationModel = configurationAccessor.createConfiguration(key, context, valueMap.values());
         }
         FieldModel convertedFieldModel = configurationFieldModelConverter.convertToFieldModel(configurationModel);
-        return apiAction.afterUpdateAction(convertedFieldModel);
+        return apiAction.afterUpdateAction(beforeUpdateAction, convertedFieldModel);
```

Comment:
The name of beforeUpdateAction confused me here, can we change it to something like, beforeUpdateActionResult or something similar?